proxa/Docusign-Integration
# Added OAuth2 authentication

Test procedures:
1. gradle bR
2. navigate to http://localhost:8080/login
3. authenticate with docusign
4. you should be redirected to the main page (it looks weird though, I know)
